pkgs/development/tools/misc/lsof/default.nix

```diff
@@ -1,9 +1,12 @@
 { stdenv, fetchurl }:
-
+let pkgs = import <nixpkgs> {};
```

Comment:
Thanks, @adnelson. You're absolutely right. This is my first PR to nixpkgs. Please consider it a beginner's mistake :)